checker/metrics/conversion/trigger_metrics_test.go

```diff
@@ -276,11 +281,46 @@ func TestTriggerMetrics_Populate(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "in last check exist metric with name of alone metric",
+			m: TriggerMetrics{
```

Comment:
It seems to me that even here it is worth using the full name of the variable. Maybe it is worth changing the structure?

Reply:
It's an auto-generated test. The name of structure attribute here taken from the name of tested function receiver.